apps/tangle-dapp/data/liquidStaking/useTokenUnlockDurations.ts

```diff
@@ -0,0 +1,29 @@
+// This will override global types and provide type definitions for
+// the `lstMinting` pallet.
+import '@webb-tools/tangle-restaking-types';
```

Comment:
Not sure if we can only import this one time for the whole project, maybe in `types/index.ts`

Reply:
I think it's better to limit the scope where these types are imported so that Tangle's types and the parachain's don't overlap or override each other.